target_postgres/target.py

```diff
@@ -138,6 +167,70 @@ def __init__(
                 + "for more information."
             ),
         ),
+        th.Property(
+            "ssl_enable",
+            th.BooleanType,
+            default=False,
+            description=(
+                "Whether or not to use ssl to verify the server's identity. Use"
+                + " ssl_certificate_authority and ssl_mode for further customization."
+                + " To use SSL to authenticate yourself to the server, use"
+                + " ssl_client_certificate_enable instead."
+                + " Note if sqlalchemy_url is set this will be ignored."
+            ),
+        ),
+        th.Property(
+            "ssl_client_certificate_enable",
+            th.BooleanType,
+            default=False,
+            description=(
+                "Whether or not to provide client-side certificates as a method of"
+                + " authentication to the server. Use ssl_certificate and"
+                + " ssl_private_key for further customization. To use SSL to verify"
```

Comment:
The current way it's set up, there are two settings to enable/disable the two types of SSL—`ssl_enable` and `ssl_client_certificate_enable`—with each having two settings that allow further customization: `ssl_mode` and `ssl_certificate_authority` for the former, `ssl_certificate` and `ssl_private_key` for the latter.

If you think it's desirable, I could change it so that the two enable/disable settings are removed, and the use or nonuse of SSL is determined by whether the other settings have values or not.

Reply:
I think I get it now but it's really confusing to me.  In the readme can we add a section on SSL and describe all of this in more detail.

Also if `ssl_client_certificate_enable` is only used with `ssl_certificate` and `ssl_private_key` then should we name it `ssl_client_certificate` and `ssl_client_private_key` , It's also surprising to me 

I think https://www.postgresql.org/docs/current/libpq-ssl.html does a good job explaining all of this so maybe a link to here would help too?

Reply:
@sebastianswms I think we still need this as well